test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -113,7 +115,7 @@ private String nowPlus(int minutes) {
       case MYSQL:
         return "date_add(current_timestamp(), interval " + minutes + " minute)";
       case ORACLE:
-        // TODO(sv): minute_add???
+        return "(select sysdate + interval '" + minutes + "' minute from dual)";
```

Comment:
Wouldn't it be: return "(CURRENT_TIMESTAMP + interval '" + minutes + "' minute)"